processor/processor.go

```diff
@@ -365,6 +369,11 @@ func (proc *HandleT) Setup(backendConfig backendconfig.BackendConfig, gatewayDB
 	proc.statBatchDestNumOutputEvents = proc.stats.NewTaggedStat("processor.num_output_events", stats.CountType, stats.Tags{
 		"module": "batch_router",
 	})
+	proc.DBReadThroughput = proc.stats.NewStat("processor.db_read_throughput", stats.CountType)
```

Comment:
Shouldn't all throughputs be calculated as `Gauge`s instead of `Counter`s?
What's the point of having a counter increased by a value (throughput) every time we calculate this value?

Reply:
any comment @saurav-malani ?